packages/legend-query-builder/src/stores/QueryBuilderState.ts

```diff
@@ -554,10 +555,14 @@ export abstract class QueryBuilderState implements CommandRegistrar {
     }
   }
 
-  get validationIssues(): string[] | undefined {
+  get validationIssues(): string[] {
     return this.fetchStructureState.implementation.validationIssues;
   }
 
+  get allValidationIssues(): string[] {
```

Comment:
why do we need 2 on the queryBuilderState. 
It should read from everything 
`return this.fetchStructureState.implementation.allValidationIssues`